### ISSUE TITLE
episodeguides tag in nfo

### ISSUE THREAD

**Issue by kebogaer** [pymedusa/Medusa#6269]
I first added this to Kodi forum, but of course I violated 36 rules by doing so, so I am adding it here again:
I don't know whether this came from recently playing around with metadata from Medusa or not, but here is what I experienced:
A while ago I enabled the flag in Medusa to create Metadata, don't know why I did this. I think in my head it would make it easier that when my MariaDB crashes a rebuild would be a lot easier with all of this metadata inside. Not a problem, until yesterday I created a new show in Medusa, and it seems to make the tvshow.nfo without the episodeguides part.
Now when I updated my library, it updated the DB with the show's entry, but it did not scrape the separate episodes.

I solved this by adding the episodeguide URL manually in the nfo file, but here's my question:

-Is this behavior what I experienced normal?
-If yes, why isn't KODI adding this information in the nfo itself?
-I saw that the TVDB scraping is not working as it should (by not using a URL), does that have anything to do with it?

**Comment by kebogaer** [comment]
noone?

**Comment by p0psicles** [comment]
I saw your issue. But didn't have the time to react properly. So you'll need to have some patience

**Comment by p0psicles** [comment]
@kebogaer we do add the episodeguide tag to the tvshow.nfo.
For example a recent show:
```
<episodeguide>
    <url cache="auth.json" post="yes">https://api.thetvdb.com/login?{"apikey":"0629B785CE550C8D","id":354171}|Content-Type=application/json</url>
  </episodeguide>
```

You have to enable this in config > postprocessing > metadata and check `Show metadata` for KODI12+

**Comment by p0psicles** [comment]
It can be you have created the tvshow.nfo in the past. And it still has a reference to the old tvdb apiv1 episodeguide url. Then your best of just deleting the tvshow.nfo. It will be generated again.

**Comment by kebogaer** [comment]
Thx P0psicles
I think I just added the Kodi, not the Kodi12+ item.
So next time it happens, I will just add KODI12+.
I did remove the tvshow.nfo file though, it didnt seem to be recreated, when is this recreation done?

Why are people actually adding this tvshow.nfo stuff directly from Medusa, what are the use cases?
Is it correct to assume that rebuilding your kodi database is quicker this way when you have all of this info next to the files?

**Comment by p0psicles** [comment]
Medusa automates this action for you. And you it saves time parsing in Kodi as it saves a couple of api requests per show. I'll close it for now.

The nfo file is recreated when you force rescan files, force full update or let it auto update during one of the nightly show updates.

**Comment by gerbil42** [comment]
Is the <episodeguide> chunk the key piece for Add Existing Show to pre-identify the show from tvshow.nfo?

I recently had an external array go offline unexpectedly and that resulted in Medusa database corruption so I started over and used Add Existing Show from my root directory.

Most of the shows were not pre-identified and I'm trying to determine why.  I confirmed that Kodi 12+ Show Metadata is checked, but neither Scan Files or Force Full Update added the episodeguide chunk of data.
